HomeToGather/HomeToGather/View/Create/Last/ColorPickerView.swift

```diff
@@ -0,0 +1,47 @@
+
+import Foundation
+
+import SwiftUI
+
+struct ColorPickerView: View {
+    var colors = ["red","blue","green","yellow"]
+    
+    @State var selectedColor = "red"
+    
+    var body: some View {
+        ZStack {
+            Color.backgroundColor.ignoresSafeArea()
+            
+            VStack(spacing:0){
+                
+                ScrollView(.horizontal, showsIndicators: false, content: {
+                    HStack {
+                        ForEach(colors,id:\.self) { color in
+                            ColorView(color: color, selectedColor: $selectedColor)
+                                .onTapGesture {
+                                    selectedColor = color
+                                }
+                        }
+                    }
+                }).background(Color.backgroundColor)
+                //
+                ticketView(isTicketGesture: false, color: $selectedColor)
+                    .background(Color.clear)
+                
+                Button {
+                    
```

Comment:
Action은 추후에 추가될 예정인가요 ?
```suggestion
                    // TODO: Action 추가
```